pep/app/session-stores/application.js

```diff
@@ -2,7 +2,7 @@ import CookieStore from 'ember-simple-auth/session-stores/cookie';
 import ENV from 'pep/config/environment';
 
 export default CookieStore.extend({
-    cookieName: 'pepweb_session',
+    cookieName: 'pep_session',
```

Comment:
this was changed again? 🤔

Reply:
? just  curious as to why this is showing up as a diff again, as i thought this change was already in master from the prior PR.

Reply:
@billdami Probably because I haven't merged in master. But since there are no conflicts it should just merge together